Common/Kernel/LoggingEngine/LoggerProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace Kernel.LoggingEngine
+{
+    public class LoggerProvider: ILoggerProvider
+    {
+        private readonly IServiceProvider _provider;
```

Comment:
How about change var name accordin to code convention

Reply:
Fixed